### PR TITLE
Support multiple topics via ros2 topic hz.

### PR DESCRIPTION
**Pull request by fujitatomoya**
replaces https://github.com/ros2/ros2cli/pull/831 and closes https://github.com/ros2/ros2cli/issues/830

Note: after the merge, we need to update the documentation

### CODE REVIEW

File: ros2topic/ros2topic/verb/hz.py [hunk 1]
```diff
@@ -225,43 +225,98 @@ def get_hz(self, topic=None):
...
+    for topic in topics:
+        msg_class = get_msg_class(
+            node, topic, blocking=True, include_hidden_topics=True)
 
-    if msg_class is None:
-        node.destroy_node()
-        return
+        if msg_class is None:
+            node.destroy_node()
+            return
```
Comment (fujitatomoya) [inline ros2topic/ros2topic/verb/hz.py:298]: I believe that what we can do better here is to skip the unknown message classes with printing the warning message for user, and proceed to measure the frequency with found ones.

**Comment by TannerGilbert** [comment]
Thanks for the quick reply on the other PR. I have one more question. Will changes like these only be merged into rolling or will there also be backports to older still supported versions like Humble?

**Comment by fujitatomoya** [comment]
@TannerGilbert 

basically we enforce API compatibility for released distribution, see more details for https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Developer-Guide.html#versioning. and this PR breaks API combability including behavior change, so we would not backport the fix to released distributions.

If we want to backport, what i would suggest is that,

1. merge this fix into rolling 1st, then wait for a couple of weeks as soak time to make sure nothing happens in the main.
2. and then, consider backport fix with API compatibility to jazzy, iron and humble. (each distro could have base code difference)

CC: @clalancette

File: ros2topic/ros2topic/verb/hz.py (hunk 1 above)
Comment (fujitatomoya) [reply ros2topic/ros2topic/verb/hz.py]: fixed with https://github.com/ros2/ros2cli/pull/929/commits/0d066fd18b55d38a27d000f4404cf86458e99258

**Comment by fujitatomoya** [comment]
@ahcorde @clalancette @sloretz i would like to request the review.

**Pull request merged**